src/App.module.css

```diff
@@ -59,7 +59,11 @@
 }
 
 .options {
-  display: flex;
+  /* display: flex; */
```

Comment:
We don't intend to keep comments. Once you decide to use another approach it would be great to remove old code.